knowledge-base/grid-customize-delete-confirmation-dialog.md

```diff
@@ -63,70 +64,120 @@ The keys for the elements of the built-in Delete Confirmation Dialog are:
 
 Use [Predefined Confirm Dialog]({%slug dialog-predefined%}#confirm) with the desired custom text. Additionally, you may get the details for the current item and add them to the text:
 
-* Handle the [`OnDelete`]({%slug grid-events%}#cud-events) event of the Grid
-* Display the Predefined Dialog in the `OnDelete` handler
-* Cancel the event or proceed with the `OnDelete` logic depending on the user choice
+* Use the Grid `Class` parameter to set a `z-index` style, which is lower than the default Dialog `z-index` of 10,000.
+* Handle the [`OnDelete`]({%slug grid-events%}#cud-events) event of the Grid.
+* Display the predefined Dialog in the `OnDelete` handler.
+* Cancel the event or proceed with the `OnDelete` logic depending on the user choice.
+* The same approach is applicable for the `OnCreate` and `OnUpdate` events.
```

Comment:
I recommend going with `applicable to`. [Source 1](https://twominenglish.com/applicable-to-or-for/) and [source 2](https://textranch.com/c/applicable-for-or-applicable-to/).